src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/index.tsx

```diff
@@ -86,15 +64,18 @@ const SearchTabPanel = () => {
         handleQuerySubmit({queryString: ev.target.value});
     };
 
-    const handleQueryOptionsChange = (
-        _: React.MouseEvent<HTMLElement>,
-        newOptions: QUERY_OPTION[]
-    ) => {
-        setQueryOptions(newOptions);
+    const handleCaseSensitivityButtonClick = () => {
+        handleQuerySubmit({
+            isCaseSensitive: isCaseSensitive,
+        });
+        setIsCaseSensitive(!isCaseSensitive);
+    };
+
+    const handleRegexButtonClick = () => {
         handleQuerySubmit({
-            isCaseSensitive: getIsCaseSensitive(newOptions),
-            isRegex: getIsRegex(newOptions),
+            isRegex: !isRegex,
         });
```

Comment:
ditto. let's make it a single line:
```suggestion
        handleQuerySubmit({isRegex: !isRegex});
```